docs/monster-ui/countrySelector().md

```diff
@@ -0,0 +1,31 @@
+title: countrySelector()
+
+# monster.ui.countrySelector()
+
+## Syntax
+```javascript
+monster.ui.countrySelector($target[, selectedValues, options]);
```

Comment:
I'd rather you have `selectedValues` defined as part of `options` to avoid having to pass down `undefined` as a placeholder parameter:
```js
monster.ui.countrySelector($target, undefined, {...});
```
Something like `monster.ui.countrySelector($target, { selectedValues: [...], options: {...}})` looks more practical, let me know what you think.

Reply:
I agree with you. Do you think it would be OK to include the `$target` parameter in that object too? Or is it better to leave it as a separated parameter, because it is required?

Reply:
I'd leave it as its own parameter since it is required, plus it's a pattern we use throughout most of our UI helpers (the newer ones at least).